redish/libs/redish-shared/domain/src/lib/error/redish-error.ts

```diff
@@ -14,6 +14,8 @@ export class RedishError {
     public cause: unknown = null
   ) {}
 
+  public static Unknown = () => new RedishError(-1, 'Unknown error');
```

Comment:
When are we actually using an unknown error? It's okay for development, but ideally, we should remove this before before going to production as we should be able to handle all errors by that point

Reply:
In authentication-api.service.ts I tried to handle an error and extract the redish error from backend. But if that does not work, there was an unknown error, right? 🤔
```
} catch (error: unknown) {
      if (axios.isAxiosError<RedishError>(error)) {
        console.error('in api service', error.response?.data);
        return error.response?.data ?? RedishError.Unknown();
      }
      return RedishError.Unknown();
    }
```

will be moved to the handle error provider, once implemented